Formula/grin-wallet.rb

```diff
@@ -14,9 +14,24 @@ class GrinWallet < Formula
   depends_on "llvm" => :build # for libclang
```

Comment:
macOS provides `libclang` (on Big Sur, at least). I wonder if we can do without this dependency. Doesn't have to be done in this PR, though.

Reply:
Updated to `uses_from_macos "llvm" => :build`. Mojave just passed, so hopefully will be fine with newer Clang too.